src/main/include/RobotMap.h

```diff
@@ -28,6 +28,8 @@ constexpr int kIDCargoRoller {9};
 
 //Hatch Motors
 constexpr int kIDHatchPivot {10};
+constexpr int kIDHatchPuncherForward {2};
+constexpr int kIDHatchPuncherReverse {6};
```

Comment:
`kIDHatchPuncherForward` and `kIDHatchPuncherReverse` are already defined. Remove these lines.